cask_index.html

```diff
@@ -11,9 +11,8 @@ <h2><a href="{{ site.baseurl }}/api/cask.json"><code>/api/cask.json</code> (JSON
     {%- assign sorted_casks = site.data.cask | sort -%}
     {%- for c in sorted_casks -%}
     <tr>
-        {%- assign token = c[0] -%}
         {%- assign cask = c[1] -%}
-        {%- include cask.html token=token cask=cask -%}
```

Comment:
As above: ideally, rename this to "key" or something to suggest it's not always the same as the token value.